plugins/node/opentelemetry-instrumentation-fastify/test/instrumentation.test.ts

```diff
@@ -86,15 +86,15 @@ describe('fastify', () => {
   let app: FastifyInstance;
 
   async function startServer(): Promise<void> {
-    const address = await app.listen(0);
+    const address = await app.listen({ port: 0 });
```

Comment:
Fastify will deprecate the prevent listen contact in V5, and is moving to this contract.